frontend-web/webclient/app/UCloud/FilesApi.tsx

```diff
@@ -123,19 +126,22 @@ const FileSensitivityVersion = "1.0.0";
 const FileSensitivityNamespace = "sensitivity";
 type SensitivityLevel = | "PRIVATE" | "SENSITIVE" | "CONFIDENTIAL";
 let sensitivityTemplateId = "";
-function findSensitivity(file: UFile): SensitivityLevel {
+function findSensitivityWithFallback(file: UFile): SensitivityLevel {
+    // TODO(Jonas): This assumes an inherit case would be 'Private', work needs to be done for backend.
+    // TODO(Jonas): The type is not correct if this needs to be cast for this to work
```

Comment:
Not sure I understand these comments. `PRIVATE` is the correct default.

Reply:
I think the first one was in reference to have "INHERIT" as an explicit option in the backend.

The second line is in reference to a `as any`-cast that's was corrected.